src/item.cpp

```diff
@@ -8263,7 +8275,7 @@ bool item::process( player *carrier, const tripoint &pos, bool activate,
         return false;
     }
 
-    if( faults.count( fault_gun_blackpowder ) ) {
+    if ( faults.count(fault_gun_blackpowder) ) {
```

Comment:
```suggestion
    if( faults.count(fault_gun_blackpowder) ) {
```

Reply:
What's the point of this change? I thought I astyled already. Or maybe I didn't?